### DISCUSSION TITLE
Modify individual camera settings

### DISCUSSION THREAD

**Discussion by DarienRourke** [dgreif/ring#673] [Q&A] [answered]
I would like to have the option to treat motion detection as doorbell events for only certain cameras (for example, the ones around my house but not my driveway camera) so I get the popups on my Apple TV (Very nice work, btw, works great!).  Is there a way to disable motion alerts for certain cameras?  If not, how do I delete my Driverway camera entirely from Hoobs but keep the rest of my cameras?
Thanks!

**Comment by dgreif** [comment, marked as answer]
From what I've heard, motion activated live streams should be part of the iOS 15 release. When that feature comes out, I'll be removing the sendDoorbellMotionNotificationsToTv feature. This feature was a bit of a hack, and not something I plan to support long term.

If you would like to remove an individual camera as a workaround, you can use the `hideDeviceIds` option.  Device ids are logged out for each device when you reboot homebridge.
